### PR TITLE
DEVX-1660: Implement common retry function for deterministic wait for…

### PR DESCRIPTION
**Pull request by ybyzek**
… services

### CODE REVIEW

**Comment by ybyzek** [comment]
@rspurgeon given the timing issues you reported yesterday, I hope this resolves it in a deterministic way

File: utils/helper.sh [hunk 1]
```diff
@@ -679,6 +679,68 @@ function wait_for_connector_up() {
...
+  fi
+  return 0
+}
+
+check_topic_exists() {
+  containerName=$1
+  zkConn=$2
+  topic=$3
+
+  docker-compose exec $containerName kafka-topics --zookeeper $zkConn --describe --topic $topic >/dev/null
```
Comment (rspurgeon) [inline utils/helper.sh]: I think we should be use the `--bootstrap-server` version of this command

`kafka-topics --bootstrap-server broker-dc1:9091 --describe --topic topic1` for example

Comment (ybyzek) [reply utils/helper.sh]: @rspurgeon 💯 and fixed

**Review by rspurgeon — APPROVED** [review]
LGTM

**Pull request merged**